sql/src/main/java/io/crate/planner/Planner.java

```diff
@@ -108,8 +115,17 @@ private static int finalLimit(@Nullable Integer queryLimit, int softLimit) {
             return queryLimit;
         }
 
+        private <T> T toPOJO(Symbol symbol, DataType type) {
```

Comment:
since only Integer returntype is used, would it be more efficient to not use convert? however dunno if this is possible, but i think it should right? convert creates a new literal if the type does not match